src/Controllers/API.php

```diff
@@ -336,7 +112,7 @@ protected function renameResource($data)
                                 continue;
                             }
 
-                            $newJSON = $this->getResource($data['to_public_id'], $oldJSON->resource_type);
+                            $newJSON = $this->getResourceData($data['to_public_id'], $oldJSON->resource_type);
```

Comment:
If you make the below changes then you would be able to use `Helper::process_resource()` directly

Reply:
Can't extend statics, won't fix.